test_util/test_upgrade_vpc.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+import logging
+import os
+import random
+import string
+import sys
+
+import test_util.aws
+import test_util.cluster
+
+
+logging.basicConfig(format='[%(asctime)s|%(name)s|%(levelname)s]: %(message)s', level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+
+def main():
+    num_masters = 3
+    num_agents = 2
+    num_public_agents = 1
+    stack_name = 'upgrade-test-' + ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
+
+    stable_installer_url = os.environ['STABLE_INSTALLER_URL']
+    installer_url = os.environ['INSTALLER_URL']
+
+    vpc, ssh_info = test_util.aws.VpcCfStack.create(
```

Comment:
nit: a little obnoxious this code needs to be largely replicated between this and test-vpc, but works for now. Hopefully before too much longer the bigger refactors will come in and fix this.